web/tests/functional/diff_remote/test_diff_remote.py

```diff
@@ -4,20 +4,27 @@
 #   This file is distributed under the University of Illinois Open Source
 #   License. See LICENSE.TXT for details.
 # -----------------------------------------------------------------------------
+
+"""diff_remote function test.
+
+Test the compraison of two remote (in the database) runs.
 """
-Diff feature tests. Comparing results from two runs.
-"""
+
 from __future__ import print_function
 from __future__ import division
 from __future__ import absolute_import
 
-import logging
 import os
 import re
 import subprocess
 import unittest
 
-from codeCheckerDBAccess_v6.ttypes import *
+from codeCheckerDBAccess_v6.ttypes import CompareData
```

Comment:
It can be written in one line:
```suggestion
from codeCheckerDBAccess_v6.ttypes import CompareData, DiffType, \
    ReportFilter, Severity, ReviewStatus, RunHistoryFilter
```